components/Carousel/Carousel.js

```diff
@@ -17,3 +17,41 @@
     <div class="right-button"> > </div>
   </div>
 */
+const carouselContainer = document.querySelector(".carousel-container")
+console.log(carouselContainer)
+
+carouselContainer.appendChild(createCarousel())
+
+
+function createCarousel(){
+  //create the elements
+  const carousel = document.createElement('div')
+  const leftButton = document.createElement('div')
+  const imgMountains = document.createElement('img')
+  const imgComputer = document.createElement('img')
+  const imgTree = document.createElement('img')
+  const imgTurnTable = document.createElement('img')
+  const rightButton = document.createElement('div')
+
+  //set the style
+  carousel.classList.add('carousel')
+  leftButton.classList.add('left-button')
+  rightButton.classList.add('right-button')
+
+  //set the content
+  imgMountains.src = "./assets/carousel/mountains.jpeg"
+  imgComputer.src = "./assets/carousel/computer.jpeg"
+  imgTree.src = "./assets/carousel/trees.jpeg"
+  imgTurnTable.src = "./assets/carousel/turntable.jpeg"
+
+  //put together
+  carousel.appendChild(leftButton)
+  carousel.appendChild(imgMountains)
+  carousel.appendChild(imgComputer)
+  carousel.appendChild(imgTree)
+  carousel.appendChild(imgTurnTable)
+  carousel.appendChild(rightButton)
+
+  return Carousel
```

Comment:
**Typo** `Carousel` instead of `carousel`. At the end of your functional component you have to return the "main" container of the elements you created.